runtime/builder.go

```diff
@@ -29,15 +29,14 @@ func (a *AppBuilder) Build(
 	logger log.Logger,
 	db dbm.DB,
 	traceStore io.Writer,
-	msgServiceRouter *baseapp.MsgServiceRouter,
```

Comment:
No need to resolve this and then pass this parameter back into `Build`. `runtime` created it in the first place